### DISCUSSION TITLE
[✨] Ability to automatically add fmatic workflows to `.gitignore`

### DISCUSSION THREAD

**Discussion by ziyadvf** [FlutterMatic/desktop#55] [Ideas]
**Is your feature request related to a problem? Please describe.**
It is annoying having to manually add `fmatic` folder to `.gitignore` for each individual project

**Describe the solution you'd like**
Option to add folder to project `.gitignore`

**Describe alternatives you've considered**
To do it manually.

**Comment by yahu1031** [comment]
Can you describe it more please.
